src/general/OptionsMenu.cs

```diff
@@ -263,6 +267,7 @@ public override void _Ready()
         performanceTab = GetNode<Control>(PerformanceTabPath);
         cloudInterval = GetNode<OptionButton>(CloudIntervalPath);
         cloudResolution = GetNode<OptionButton>(CloudResolutionPath);
+        runAutoEvo = GetNode<CheckBox>(RunAutoEvoPath);
```

Comment:
Could you rename this to include that this is the "during gameplay" running of auto-evo and *not* entirely turning it off?